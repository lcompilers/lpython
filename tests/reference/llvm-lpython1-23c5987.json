{
    "basename": "llvm-lpython1-23c5987",
    "cmd": "lpython --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/lpython1.py",
    "infile_hash": "195a467b66705e0f2638aee8c8a6e9145112b2964e05896c328e0d86",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-lpython1-23c5987.stdout",
    "stdout_hash": "de8886bdb5b86970a042b8ee00f309cf6bcf9e3e192cb4875ab3b2b0",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}