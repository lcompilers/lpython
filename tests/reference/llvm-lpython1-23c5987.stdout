; ModuleID = 'LFortran'
source_filename = "LFortran"

define i32 @main(i32 %0, i8** %1) {
.entry:
  call void @_lpython_call_initial_functions(i32 %0, i8** %1)
  ret i32 0
}

declare void @_lpython_call_initial_functions(i32, i8**)
